functions/actions/version.js

```diff
@@ -0,0 +1,24 @@
+const mustache = require('mustache');
+
+const dialog = require('../dialog');
+const versionStrings = require('../strings').intents.version;
+const {getLastReprompt, getLastSuggestions} = require('../state/dialog');
+
+let versionPackage = require('../package.json');
+
+/**
+ * handle version intent
+ *
+ * @param app
+ */
+function handler (app) {
+  let reprompt = getLastReprompt;
```

Comment:
I don't think you need it here and well as suggestions

Reply:
I was considering that reprompt shouldn't be modified, if user asks for version?

Reply:
It could make sense but even in this case you missed `(app)` here

Reply:
I will correct it.